src/main/java/com/tstepnik/planner/service/TaskService.java

```diff
@@ -0,0 +1,63 @@
+package com.tstepnik.planner.service;
+
+import com.tstepnik.planner.domain.Importance;
+import com.tstepnik.planner.domain.Task;
+import com.tstepnik.planner.repository.TaskRepository;
+import com.tstepnik.planner.security.auth.CustomUserDetails;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.stereotype.Service;
+
+import java.security.Principal;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TaskService {
+
+    private final Importance DEFAULT_IMPORTANCE = Importance.INCIDENTAL;
+
+    private final TaskRepository taskRepository;
+
+    public TaskService(TaskRepository taskRepository) {
+        this.taskRepository = taskRepository;
+    }
+
+    public List<Task> findAll() {
+        return taskRepository.findAll();
+    }
+    public Optional<Task> getTask(Long id){
```

Comment:
This should be named findById. We can talk about difference between find and get on the call